students/josh_embury/lesson03/mailroom.py

```diff
@@ -0,0 +1,79 @@
+lst_donor_table = [
+['Henry Michalson',10,500,25],
```

Comment:
Need spaces after commas, here and below in each list element

Reply:
updated